classes/helper/HelperCalendar.php

```diff
@@ -145,10 +145,10 @@ public function getDateFormat()
         return $this->_date_format;
     }
 
-    public function setDateFrom($value = '')
+    public function setDateFrom($value)
```

Comment:
```suggestion
    public function setDateFrom($value = '')
```
That was initial value